### DISCUSSION TITLE
Difficulty running Bottom-Up id

### DISCUSSION THREAD

**Discussion by tasnimtazzy** [talmolab/sleap#1384] [Help!] [answered]
Hello! I am trying to run training on a video using the bottom-up id training model, but keep receiving the following error message: 

ValueError: test_size=1 should be either positive and smaller than the number of samples 0 or a float in the (0, 1) range. 

I have already tried to change the input scaling from 1 down to 0.5 and 0.25, but the same error message continues to pop up. Any help would be appreciated.

**Comment by roomrys** [comment, marked as answer]
Hi @tasnimtazzy,

It seems that SLEAP believes you have no annotations (or "samples") in your project.... I am just verifying that you are running training on an slp file that has some labels (not just in the video file)? Also, if you are using a supervised ID, you will need to make sure that you label the ID (the track) of the instance/animal (select the instance, navigate to the track menu on the bar up top, and set the track).

Let me know if this helps!

Thanks,
Liezl

**Reply by tasnimtazzy** [reply]
Thank you for the reply! 

I am running the training on an slp file with about 45 labeled frames, but we did not set the track of the instance before. Should we set all of the labeled frames to the same track? Thank you! 

UPDATE: I set all of the labeled frames to the same track, and I was able to begin training on bottom-up id. Thank you so much again!
